### PR TITLE
feat(stories): stories results section styling

### PR DESCRIPTION
**Pull request by runrog**
Opening new PR as https://github.com/rackerlabs/zoolander/pull/497 was merged too early

See locally here
http://localhost:9000/derek/view-templates/stories/stories

### CODE REVIEW

**Comment by runrog** [comment]
Fix has been submitted for the inline image issue:
- removed the :before selector and replaced with a div so that we can inline an image.

seen here: http://localhost:9000/derek/view-templates/stories/stories

File: styleguide/derek/view-templates/stories/stories.ejs [hunk 1]
```diff
@@ -107,10 +110,14 @@ var featQuotes = [
         </div>
         <% } %>
         <div class="row">
-          <ul class="pager">
-            <li class="pager-previous first"><a href="javascript:void(0);">prev</a></li>
+          <ul class="pager rsFilter-storiesPager">
```
Comment (writes) [inline styleguide/derek/view-templates/stories/stories.ejs]: @runrog this class `rsFilter-storiesPager` should be in a wrapper div. We don't want to add custom classes to the pager itself.

File: styleguide/derek/view-templates/stories/stories.ejs [hunk 2]
```diff
@@ -107,11 +110,13 @@ var featQuotes = [
         </div>
         <% } %>
         <div class="row">
-          <ul class="pager">
-            <li class="pager-previous first"><a href="javascript:void(0);">prev</a></li>
-            <li class="pager-current">2 of 10</li>
-            <li class="pager-next last"><a href="javascript:void(0);">next</a></li>
-          </ul>
+          <div class="rsFilter-storiesPager">
```
Comment (runrog) [inline styleguide/derek/view-templates/stories/stories.ejs]: @writes that work for you?

**Comment by writes** [comment]
👍 LGTM

**Comment by WidgetsBurritos** [comment]
[run ci]

**Pull request merged**